test/11_array.jl

```diff
@@ -650,4 +650,23 @@ end
     @test unique(x) == collect(1:10)
 end
 
+@testset "repeat" begin
```

Comment:
Can you put this in 13_arraycommon.jl instead? It would also be nice to add an array with `missing` to the list for `a`.

Reply:
moved